client/my-sites/earn/memberships/products-list.tsx

```diff
@@ -46,9 +45,8 @@ function ProductsList() {
 	const features = useSelector( ( state ) => getFeaturesBySiteId( state, site?.ID ) );
 	const hasLoadedFeatures = features?.active.length > 0;
 	const products: Product[] = useSelector( ( state ) => getProductsForSiteId( state, site?.ID ) );
-	const connectedAccountId = useSelector( ( state ) =>
-		getConnectedAccountIdForSiteId( state, site?.ID )
-	);
```

Comment:
We no longer make the display of the product list conditional on connectAccountId (ie stripe being connected).